source/_integrations/thread.markdown

```diff
@@ -155,9 +153,12 @@ Note: To add Matter devices to this Thread network, an Android phone is required
 
 3. Before you can add Matter-based Thread devices, your phone needs to know the credentials of your newly created Thread network.
    - To share the credentials with your Android phone, open the Home Assistant Companion app.
-   - In the Companion app, go to **Settings** > **Companion app** > **Troubleshooting**, then select **Sync Thread credentials**.
-   - Follow the instructions on screen.
-   - **Result**: You will see a confirmation stating that Thread credentials from Home Assistant have been added to this device.
+     - In the Companion app, go to **Settings** > **Companion app** > **Troubleshooting**, then select **Sync Thread credentials**.
+     - Follow the instructions on screen.
+     - **Result**: You will see a confirmation stating that Thread credentials from Home Assistant have been added to this device.
+   - To share the credentials with your iPhone, open the Home Assistant Companion app.
+     - Go to {% my integrations title="**Settings** > **Devices & Services**" %}, select the **Thread** integration.
+     - Below the border router, select **Send credentials to phone**.
```

Comment:
It should only appear on the preferred border router network, so something like:

```suggestion
     - At the bottom of the preferred network box, select **Send credentials to phone**.
```